src/_includes/components/avatars.css

```diff
@@ -0,0 +1,52 @@
+.icon,
+.avatar,
+blockquote img {
+	height: 1.3em;
+	width: 1.3em;
+	display: inline-block;
+	vertical-align: text-top;
+	margin-left: 2px;
+	margin-right: 6px;
```

Comment:
I'm curious: Is there a plan to move to Logical Properties (i.e. `margin-inline: 2px 6px`) somewhen?

Reply:
I don’t think we’ll do a big refactor but it’ll ship of theseus change over time